Description: disable tests with expired certificates
Author: Marc Deslauriers <marc.deslauriers@canonical.com>
Forwarded: not-needed

--- a/mysql-test/t/disabled.def
+++ b/mysql-test/t/disabled.def
@@ -19,3 +19,7 @@ explain_for_connection_rqg_json  : Bug#2
 explain_for_connection_rqg_trad : Bug#20697533 2015-03-20 erlend Fails several times each day on all platforms
 daemonize_opt : Bug#31335773 2020-05-12 Deepa Shutdown_report fails
 partition_rename_longfilename : n/a 2018-04-19 mdeslaur Fails when building locally
+main.ssl_crl_crlpath		: BUG#00000 -  expired cert
+main.ssl_ca			: BUG#00000 -  expired cert
+main.ssl_crl			: BUG#00000 -  expired cert
+main.ssl_crl_clients_valid	: BUG#00000 -  expired cert
